website/contributing/release-stable-patch.md

```diff
@@ -16,29 +23,86 @@ git pull origin <release-branch> --tags
 
 ```bash
+# once verified all the cherry-picked commits, we can push them to remote
+git push
+
 # run a script to bump the version
-# You most likely **don't** want this release marked as "latest"
+# If you are doing this patch release for the current latest minor, then mark it as "latest"
 ./scripts/bump-oss-version.js --to-version x.y.z --token <YOUR_CIRCLE_CI_TOKEN>
 ```
 
-### 4. Verify Upgrade helper is updated
+### 4. Watch CircleCI to ensure right jobs are being triggered
 
-- You should see a [new publish job complete here](https://github.com/react-native-community/rn-diff-purge/actions) which will be automatically triggered after a new tag is pushed to the `react-native` repository.
-- If not, check out the guide on [how to update Upgrade Helper](/contributing/updating-upgrade-helper).
+- Once you have run the bump script script, head to CircleCI and you should see under the releases workflow, a `prepare-package-for-release` job.
+
+  <figure>
+    <img width="400" alt="CircleCI showing publish release" src="https://user-images.githubusercontent.com/1309636/150040711-cfbc2fe3-91eb-42b9-bd06-de2aa7fb94ea.png"/>
+    <figcaption>CircleCI showing publish release.</figcaption>
+  </figure>
+
+- Once complete you should be able to run `npm view react-native` and verify that under the `latest` tag, the version is the expected release version.
+
+  ```bash
+  npm view react-native
+  ...
+  dist-tags:
+  latest: 0.Y.Z            next: 0.Y.0-rc.X         nightly: 0.0.0-f617e022c
+  ```
+
+### 5. Create a PR of the changelog using the generator
+
+To generate the changelog, we rely on a dedicated tool called [`@rnx-kit/rn-changelog-generator`](https://github.com/microsoft/rnx-kit/tree/main/incubator/rn-changelog-generator) that will parse the custom changelog messages that contributors write in their PRs.
+
+```bash
+# Run following with the stable release as base, and your rc.0 version
+npx @rnx-kit/rn-changelog-generator --base v[LATEST_STABLE]--compare v[YOUR_NEW_MINOR] \
+--repo ~/react-native --changelog ~/react-native/CHANGELOG.md
+
+# example against 0.68.2 and 0.68.3
+npx @rnx-kit/rn-changelog-generator --base v0.68.2 --compare v0.68.3 \
+--repo ~/react-native --changelog ~/react-native/CHANGELOG.md
+```
+
+Create a pull request of this change to `react-native` repo and add the `Changelog` label.
+
+### 6. Create GitHub Release
+
+Use template below for the GitHub Release:
+
+```markdown
+<!-- Template for a stable patch -->
+
+<!-- You can run `npx @rnx-kit/rn-changelog-generator --base v0.66.0 --compare v0.66.1 --repo path-to-repository/react-native --changelog path-to-repository/react-native/CHANGELOG.md` to generate the markdown for the cherry-picked changes. Make sure to use the right tags in the command and create a new PR with the changes generated by the changelog generator. You can read more about the changelog generator [here](https://github.com/microsoft/rnx-kit/tree/main/incubator/rn-changelog-generator). -->
```

Comment:
Do we need this not in Release template since we include the information already in the section above?

Reply:
it's ok to leave it there - better to overcommunicate :) it's just a comment anyway